[OU-FIX] apriori: l10n_it_dichiarazione_intento renamed to

primes2h opened the pull request:
l10n_it_declaration_of_intent in 14.0

This fix is needed to properly run these migration scripts:
https://github.com/OCA/l10n-italy/tree/14.0/l10n_it_declaration_of_intent/migrations/13.0.1.0.0

primes2h committed:
[OU-FIX] apriori: l10n_it_dichiarazione_intento renamed to
l10n_it_declaration_of_intent in 14.0

StefanRijnhart reviewed (approved):
Indeed, renamed in https://github.com/OCA/l10n-italy/pull/2043. Next time please try to preserve alphabetical order in the sublist of modules per repo.

primes2h commented:
> Indeed, renamed in [OCA/l10n-italy#2043](https://github.com/OCA/l10n-italy/pull/2043). Next time please try to preserve alphabetical order in the sublist of modules per repo.

Ops, sorry for that. Fixed, thank you!

MiquelRForgeFlow commented:
/ocabot merge nobump
